packages/react-native-toast/example/.expo-shared/assets.json

```diff
@@ -0,0 +1,4 @@
+{
+  "12bb71342c6255bbf50437ec8f4441c083f47cdb74bd89160c15e4f43e52a1cb": true,
+  "40b842e832070c58deac6aa9e08fa459302ee3f9da492c7e77d93d2fbf4a56fd": true
+}
```

Comment:
What is this?

Reply:
Autogenerated by expo, can actually be in .gitignore.

On it ⌛